src/Math/NatArith.dfy

```diff
@@ -7,5 +7,14 @@ module NatArith {
     if a < b then a else b
   }
 
+  function Factorial(n: nat, start: nat := 1): nat {
```

Comment:
I am having trouble parsing the definition of this one. Is this supposed to calculate `n * (n-1) * ... * start`? If yes, I'm not sure the definition is correct.

Reply:
I added a comment